packages/css/src/components/button/button.scss

```diff
@@ -6,7 +6,7 @@
 @import "../../common/text-rendering";
 
 .ams-button {
-  border: none;
+  border: 0;
```

Comment:
I’ve researched a bit. The actual effect of `border: 0` and `border: none` for all longhand properties are the same because the browser sets all others to their initial values, and `0` and `none`  are the initial values for `-width` and `-style`, respectively.

Theoretically, there would be different results with partial cascading.

If used as a reset, I am happy with `border: 0`.

To set or override the width of a particular border, `border-width: 0` would be more precise.

Reply:
Discussed: having `border-width` doesn’t seem to have real advantages after all.

Overriding `border` with only a width doesn’t work because the shorthand had already reset `style` to `none`.

Additionally, providing a generic `border` token in every situation gives people who theme our components the most freedom to set all longhand properties to their liking.

Reply:
As discussed: `border: 0` does the same thing as `border-width: 0` in every practical sense, and having 1 way of setting this is a simpler convention than having multiple ways.